int32 sensor
int32 chip
---
bool success
